lib/src/widget.dart

```diff
@@ -16,7 +16,6 @@ import './tile/default_tile.dart';
 /// SmartSelect that allows you to easily convert your usual form selects
```

Comment:
Can we keep the previous format? since it is not effect to the package health, i don't know, for me its a bit messy to use auto format.

Reply:
Fixed. Sorry for the auto format.

Reply:
Thanks